clientEncryptionToCPKNMigrationSamples/ClientSideKeyVaultKeyToCustomerManagedKey/src/main/java/Migration.java

```diff
@@ -0,0 +1,142 @@
+                                                  String blobDecryptName,
+                                                  String encryptionScope, String path) {
+        String storageAccountUrl = "https://" + storageAccount + ".blob.core.windows.net";
+        // Creating blob client for reuploading
+        BlobClientBuilder blobClientBuilder = new BlobClientBuilder()
+                .endpoint(storageAccountUrl)
+                .credential(new StorageSharedKeyCredential(storageAccount, sharedKeyCred))
+                .containerName(containerName)
+                .encryptionScope(encryptionScope)
+                .blobName(blobDecryptName);
+        BlobClient blobClientDecrypted = blobClientBuilder.buildClient();
+
+        // Uploading file to server
+        blobClientDecrypted.uploadFromFile(path + blobDecryptName, true);
+    }
+
+    /**
+     * Cleans up temp files created during decryption
+     */
+    private static void cleanup(String blobDecryptName, String path) {
+        // Cleaning up by deleting local save of encrypted blob
+        File localFile = new File(path + blobDecryptName);
+        localFile.delete();
+    }
+
+    public static void main(String[] args) {
+        String clientId = null;
+        String clientSecret = null;
+        String tenantId = null;
+        String storageAccount = null;
+        String sharedKeyCred = null;
+        String keyVaultUrl = null;
+        String containerName = null;
+        String blobName = null;
+        String blobNameAfterMigration = null;
+        String clientSideEncryptionKeyName = null;
+        String encryptionScope = null;
+        String keyWrapAlgorithm = null;
+
+        String pathToDir = "clientEncryptionToCPKNMigrationSamples\\ClientSideKeyVaultKeyToCustomerManagedKey\\" +
+                "src\\main\\java\\exampleDataCreator\\";
+
+        // Extracting variables from config file
+        try (InputStream input = new FileInputStream(pathToDir + "app.config")) {
+            Properties prop = new Properties();
+            prop.load(input);
+            clientSecret = prop.getProperty("clientSecret");
+            clientId = prop.getProperty("clientId");
+            tenantId = prop.getProperty("tenantId");
+            storageAccount = prop.getProperty("storageAccount");
+            sharedKeyCred = prop.getProperty("sharedKeyCred");
+            keyVaultUrl = prop.getProperty("keyVaultUrl");
+            containerName = prop.getProperty("containerName");
+            blobName = prop.getProperty("blobName");
+            blobNameAfterMigration = prop.getProperty("blobNameAfterMigration");
+            clientSideEncryptionKeyName = prop.getProperty("clientSideEncryptionKeyName");
+            encryptionScope = prop.getProperty("encryptionScope");
+            keyWrapAlgorithm = prop.getProperty("keyWrapAlgorithm");
+        } catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
Please make `main(...) throws IOException` or re-throw runtime wrapping this `IOException`. It doesn't make sense to continue the program if that happens. If you rethrow JVM will automatically print the stack trace to stderr.